src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 ## Unreleased
 
-* Split up Prometheus projects based on its hosting mechanism, HttpListener and AspNetCore,
-  into their own projects and assemblies. The shared code for both hosting mechanism
-  now lives in the `OpenTelemetry.Exporter.Prometheus.Shared` project and will not
-  be released.
-  ([#3430](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3430))
+* Split up Prometheus projects based on its hosting mechanism,
+  HttpListener and AspNetCore, into their own projects
+  and assemblies.
+  ([#3507](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3507),
+  [#3503](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3503) &
```

Comment:
`&` might not be a common case (maybe it's one-off in some existing places).

I think the most commonly used approach https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md#120-rc3. So it'll look exactly like `(#3430 #3503 #3507)`.

Reply:
It looks like ascending order commonly used in this repo.